website/content/v1.11/advanced/etcd-maintenance.md

```diff
@@ -74,3 +74,114 @@ NODE         MEMBER             DB SIZE   IN USE             LEADER
 
 Regular backups of `etcd` database should be performed to ensure that the cluster can be restored in case of a failure.
 This procedure is described in the [disaster recovery]({{< relref "disaster-recovery" >}}) guide.
+
+## Downgrade v3.6 to v3.5
+
+### Server downgrade checklists
+
+Before beginning, check the etcd health and download snapshot, as described in [disaster recovery]({{< relref "disaster-recovery" >}}).
+Should something go wrong with the downgrade, it is possible to use this backup to rollback to existing etcd version.
+
+### Downgrade procedure
+
+This example shows how to downgrade an etcd in Talos cluster.
+
+#### Step 1: check downgrade requirements
+
+Is the cluster healthy and running v3.6.x?
+
+```bash
+$ talosctl -n <CP1>,<CP2>,<CP3> etcd status
+NODE         MEMBER             DB SIZE   IN USE            LEADER             RAFT INDEX   RAFT TERM   RAFT APPLIED INDEX   LEARNER   PROTOCOL   STORAGE   ERRORS
+172.20.0.4   a2b8a1f794bdd561   3.6 MB    2.2 MB (61.59%)   a49c021e76e707db   4703         2           4703                 false     3.6.4      3.6.0
+172.20.0.3   912415ee6ed360c4   3.5 MB    2.2 MB (61.88%)   a49c021e76e707db   4703         2           4703                 false     3.6.4      3.6.0
+172.20.0.2   a49c021e76e707db   3.5 MB    2.2 MB (62.06%)   a49c021e76e707db   4703         2           4703                 false     3.6.4      3.6.0
+```
+
+#### Step 2: download snapshot backup from leader
```

Comment:
Also the official style guide uses, a title case format for titling. Where all the first letters of each major word is capitalized

Reply:
Thanks, fixing!